ddp-workspace/projects/ddp-sdk/src/lib/services/languageService.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+
+@Injectable()
+export class LanguageService {
+    constructor(private translate: TranslateService) { }
+
+    public getCurrentLanguage(): string {
+        return this.translate.currentLang;
+    }
+}
```

Comment:
I decided to create `LanguageService` because in the future we will need to manage/store chosen language in the LocalStorage when we will create LanguageComponent, so it just the initial step in this direction.